botfront/imports/api/graphql/botResponses/resolvers/nlgResolver.js

```diff
@@ -0,0 +1,87 @@
+import { safeLoad } from 'js-yaml';
+import { sample } from 'lodash';
+import { newGetBotResponses } from '../mongo/botResponses';
+
+const subText = (text, slots) => {
+    // fills in {slotname} in templates
+    const slotSubs = Object.entries(slots).map(s => [`{${s[0]}}`, s[1] || '']);
```

Comment:
Create a function and substitute both calls (here and in templates.controller.js)

Reply:
Can't, since templates.controller.js is in the API package directory.